src/cmd/services/m3coordinator/main/main.go

```diff
@@ -25,7 +25,7 @@ import (
 	_ "net/http/pprof" // pprof: for debug listen server if configured
 	"os"
 
-	"github.com/m3db/m3db/src/coordinator/services/m3coordinator/server"
+	"github.com/m3db/m3db/src/query/services/m3coordinator/server"
```

Comment:
The plan is to follow up and move the package being imported here to `src/coordinator/server/...` yeah?